src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -76,7 +76,6 @@
                                  :dispatch [:onboarding-2/navigate-to-identifiers]}]
                :db             (-> db
                                    (dissoc :profile/login)
-                                   (dissoc :auth-method)
```

Comment:
does anyone know why we `dissoc` this here? 🤔 -> seems to me like we should keep it 🤷‍♂️

Reply:
We clean up the `:profile/login` and `:auth-method` keys after the successful generation of keys.

If we retain the keys, we will encounter this issue https://github.com/status-im/status-mobile/issues/15632.